shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/communication/DatabaseCommunicationEngine.java

```diff
@@ -207,7 +207,7 @@ public QueryResponseRow getQueryResponseRow() throws SQLException {
         List<QueryResponseCell> cells = new ArrayList<>(queryHeaders.size());
         boolean isBinary = isBinary();
         for (int columnIndex = 1; columnIndex <= queryHeaders.size(); columnIndex++) {
-            Object data = mergedResult.getValue(columnIndex, Object.class);
+            Object data = mergedResult.getValue(columnIndex, queryHeaders.get(columnIndex - 1).getColumnType() == 93 ? String.class : Object.class);
```

Comment:
What's mean of 93, could you define a constant for it?

Reply:
yes, i will define  a constant. 93 represents the datetime type.